R/IntakeDetailedProtein.R

```diff
@@ -2,6 +2,8 @@
 #' @description Calculates detailed per-capita (protein in grams) intake from magpie results at regional level
```

Comment:
"Calculates food-specific per-capita protein intake from magpie results in grams"